src/Components/Form/Form.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import ApiRequest from '../ApiRequest/ApiRequest';
+import './Form.scss';
+
+class Form extends Component {
+  submitForm = event => {
+    event.preventDefault();
+
+    const locationFieldValue = event.target.location.value;
+    const data = ApiRequest(locationFieldValue);
+
+    fetch(data)
+      .then(response => response.json())
+      .then(response => this.props.handleState(response))
+      .catch(error => console.error('Error:', error));
```

Comment:
you should somehow handle errors.
loging them in console is not an error handling. 

you can set in state 
errorMessage: '',

and here inside catch, if an error occurs you can set errorMessage like "There was an error while fetching data. Please refresh the page to try again."

and basing on this errorMessage prop, you can render this in render method as a message for a user